swig/CMakeLists.txt

```diff
@@ -44,11 +44,22 @@ foreach(incdir ${GNURADIO_RUNTIME_INCLUDE_DIRS})
     list(APPEND GR_SWIG_INCLUDE_DIRS ${incdir}/gnuradio/swig)
 endforeach(incdir)
 
+if(${ENABLE_SATTOOLS_ZEROMQ})
+    set(ENABLE_SATTOOLS_ZEROMQ_DEFINE_STATEMENT "#define _INCLUDE_ZMQ 1")
+else()
+    set(ENABLE_SATTOOLS_ZEROMQ_DEFINE_STATEMENT "")
+endif()
+
+configure_file(
```

Comment:
Can you use `GR_SWIG_FLAGS` instead of a template?

Reply:
It's unclear to me how to use SWIG flags to do what we want. I will need a much deeper understanding of SWIG for this. Maybe we could have it as a TODO.

Reply:
I was thinking `GR_SWIG_FLAGS=-D_INCLUDE_ZMQ` would work - swig is similar to gcc and can take preprocessor defines with the same syntax from what I understand.

If not then this approach is fine.